extensions/funding/converters.go

```diff
@@ -0,0 +1,35 @@
+package funding
+
+import (
+	"reflect"
+
+	"github.com/centrifuge/go-centrifuge/protobufs/gen/go/funding"
+)
+
+// TODO: generic? Create something like ServiceRegistry for Custom Attributes...
+func (f *Data) initFundingFromData(data *funpb.FundingData) {
```

Comment:
I think this method could be generic

Reply:
this whole file should be removed when we get rid of the protobuf :)